docs/hooks/patient-view.md

```diff
@@ -1,10 +1,13 @@
 # `patient-view`
 
+!!! info
+    This page defines a workflow [hook](../../specification/current/#hooks) for the purpose of providing clinical decision support using CDS Hooks version 1.0. This is a **build** at the level of **[Trial Use](http://hl7.org/fhir/versions.html#std-processs)**.
```

Comment:
```suggestion
    This page defines a workflow [hook](../../specification/current/#hooks) for the purpose of providing clinical decision support using CDS Hooks. This is a **build** at the level of **[Trial Use](http://hl7.org/fhir/versions.html#std-processs)**.
```

specificatioVersion is primarily defined on this same page, in the table below. Putting it here is duplicative and invites mistakes.